apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/ConnectionPoolOptions.cs

```diff
@@ -59,8 +59,11 @@ public LogLevel LogLevel
                 {
                     return (LogLevel) underlyingLevel;
                 }
-                // ReSharper disable once NotResolvedInText
-                throw new ArgumentOutOfRangeException("LogLevel");
+                else
+                {
+                    // ReSharper disable once NotResolvedInText
+                    throw new ArgumentOutOfRangeException("LogLevel");
+                }
```

Comment:
fair enough re: throwing in general during get.  though this change is just adding some braces to ensure (facepalm) you can actually get the value at all without throwing.